Library/Formula/trace2html.rb

```diff
@@ -0,0 +1,19 @@
+class Trace2html < Formula
+  desc "Utility from Google Trace Viewer to convert JSON traces to HTML"
+  homepage "https://github.com/google/trace-viewer"
+  url "https://github.com/google/trace-viewer/archive/a4c4894801935627ff412c3cc6c3ddae96019cc5.zip"
+  sha256 "b9fc71e6f1a36f91ca09c1ea26ae2c38e42d3183b6470e603d1db8b1ee6e8316"
+
+  depends_on "Python"
```

Comment:
Don't hardcode Homebrew's Python. We use the system Python on OS X versions that support >2.7. You can invoke that dependency with `depends_on :python if MacOS.version <= :snow_leopard`.